<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceClassifierDialog</class>
 <widget class="QDialog" name="FaceClassifierDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="imageLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="faceLbl">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="glassesLbl">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>370</y>
     <width>320</width>
     <height>120</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>641</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="eyeLbl">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>250</y>
     <width>320</width>
     <height>120</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonDraw">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>510</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
